api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -108,5 +134,23 @@
         return new ResponseEntity<>(null, responseHeaders, HttpStatus.ACCEPTED);
     }
 
+    /*private void activeRequestCheck() {
+        User user = userService.getCurrentUser();
+        if (jobService.getActiveJob(user) != null) {
+            throw new RuntimeException("The current user has an active job");
+        }
+    }*/
+
+    @ControllerAdvice
+    class ErrorHandler extends ResponseEntityExceptionHandler {
```

Comment:
Let's place `ErrorHandler` in its own class file. Inner classes are kind of hard to find when looking at source tree